tests/Maker/FunctionalTest.php

```diff
@@ -807,6 +807,36 @@ public function getCommandEntityTests()
             ->setRequiredPhpVersion(70100)
         ];
 
+        yield 'entity_many_to_one_self_referencing' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeEntity::class),
+            [
+                // entity class name
+                'User',
+                // field name
+                'guardian',
+                // add a relationship field
+                'relation',
+                // the target entity
+                'User',
+                // relation type
+                'ManyToOne',
+                // nullable
+                'y',
+                // do you want to generate an inverse relation? (default to yes)
+                '',
+                // field name on opposite side
+                'wards',
```

Comment:
This word isn't commonly used in English. Using `dependents` might be better (A parent/guardian refers to their children as "dependents")